### PR TITLE
Error Dialog UI improvements

### PR DESCRIPTION
**Pull request by zzori-theoriginal**
Notes:
- Didn't get rid of the close 'x' in top right corner because doing so would get rid of the entire title area and I feel like that trade of wasn't worth it
    - Also, since the title block is still there, the user can still move the window (which I know is something that we didn't want); 
    however, I got rid of the ability to resize the window
- Added the close button at the bottom of the window as requested
- Added logic for the close button functionality 

Snapshot:
errorwindow

### CODE REVIEW

**Comment by jkonicki** [comment]
Code looks good.  

For the Dialog, it should follow the look and feel of the rest of the UI (possibly a different ticket to take a look at this).  The text box should have more whitespace around it.  Make sure the text box can scroll in the case of a large error message.  

As an example, this is more like what I would expect from the error dialog look and feel.  Note the icon near the message that shows it is an error versus a warning or informational message.
image

File: WpfViews/ErrorDialogView.xaml.cs [hunk 1]
```diff
@@ -18,9 +22,17 @@ public ErrorDialogView()
...
             _viewModel = viewModel;
-
+            CloseCommand = new DelegatingCommand(CloseCommandHandler, viewModel.CanClose);
             DataContext = viewModel;
             InitializeComponent();
+
+        }
+        public void CloseCommandHandler(object sender)
+        {
+            this.Close();
```
Comment (ccheetham) [inline WpfViews/ErrorDialogView.xaml.cs:32]: `this.` should be removed.

**Review by ccheetham — CHANGES_REQUESTED** [review]

**Review by dtillman — APPROVED** [review]

File: WpfViews/ErrorDialogView.xaml.cs (hunk 1 above)
Comment (zzori-theoriginal) [reply WpfViews/ErrorDialogView.xaml.cs]: this has been removed!

**Review by dtillman — DISMISSED** [review]

**Review by ccheetham — DISMISSED** [review]

File: WpfViews/WpfViews.csproj [hunk 2]
```diff
@@ -32,6 +32,18 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="CommonServiceLocator, Version=2.0.2.0, Culture=neutral, PublicKeyToken=489b6accfaf20ef0, processorArchitecture=MSIL">
+      <HintPath>..\packages\CommonServiceLocator.2.0.2\lib\net47\CommonServiceLocator.dll</HintPath>
+    </Reference>
+    <Reference Include="GalaSoft.MvvmLight, Version=5.4.1.0, Culture=neutral, PublicKeyToken=e7570ab207bcb616, processorArchitecture=MSIL">
```
Comment (atwoosnam) [inline WpfViews/WpfViews.csproj:38]: Do we still need these extra references? I can't remember what you said they were needed for @zzori-theoriginal

**Review by atwoosnam — APPROVED** [review]

**Pull request merged**